src/scaledmap.jl

```diff
@@ -64,3 +64,6 @@ for (In, Out) in ((AbstractVector, AbstractVecOrMat),
         end
     end
 end
+
+mul!(Y::AbstractMatrix, c::Number, X::ScaledMap, a::Number, b::Number) =
+    mul!(Y, c, X.lmap, a*X.λ, b)
```

Comment:
This should perhaps be moved to the `(In, Out)`-loop above and adjusted to the situation there. Rearranging factors requires the types to belong to some commutative class like `RealOrComplex`.